sdk/aqueduct/integrations/connect_config.py

```diff
@@ -184,16 +184,20 @@ class Config:
 
 
 class AWSConfig(BaseConnectionConfig):
-    access_key_id: str
-    secret_access_key: str
-    region: str
+    access_key_id: str = ""
+    secret_access_key: str = ""
+    region: str = ""
+    config_file_path: str = ""
+    config_file_profile: str = ""
```

Comment:
Oh it would be really helpful to know what fields are required and which ones are optional here. Eg. What happens if all the fields are set? Which configuration is picked? We either set the key/secret directory or we use a profile path right? We should enforce that.

Reply:
@kenxu95 Yeah so this is enforced at the backend's Golang authentication helper. We either require all first 3 fields or the last 2, and any other possibilities are rejected. I think adding the check in the backend is more robust because it guard against both UI, SDK, and other REST calls. Should we just add comments here explaining what we allow or do you want to add the duplicate check here as well?